requirements/edx/github.in

```diff
@@ -78,8 +78,10 @@
 # Release candidates being tested.
 ##############################################################################
 
-# ... add dependencies here
-
+# https://github.com/openedx/django-pyfs/pull/169
```

Comment:
Can you remove the reference to openedx-django-pyfs in base.in?

Reply:
@viadanna There is no direct reference to openedx-django-pyfs in base.in, it is pulled by the xblock requirement.

Reply:
Alright, that's where we need to bump `openedx-django-pyfs`

Reply:
Done.